src/SingleCellAnnotator.imjoy.html

```diff
@@ -365,6 +378,18 @@
       await this.displayMap(imageData.uri);
       this.showMaskAnnotation(imageData.mask_geojson);
     },
+    // extend Image
+    async extendImage(pixels_number){
+      if (pixels_number){this.extend_pixels = pixels_number};
+      this.extend_image = true;
+      this.current_image_index = this.cached_image_data.indexOf(this.current_image_data);
+      const cell_id = this.current_image_data.cell_id;
+      const imageData = await this.fetchCellImage(cell_id)
```

Comment:
I think we don't need a global variable `this.extend_pixels`, this parameter can be passed directly to `fetchCellImage()` as an optional argument. You should try to avoid using global variables, as they will make it harder to debug issues.

And also, I think by default we should extend with a small number of pixels (e.g.: 0~20 pixels), if one switch on the adjust annotation mode, then we fetch an image with larger extent (e.g.: 100~400pixels).

Reply:
since the image shape, size differs. what if the user want to change the extend size. I set the extend_pixels to hold a place for input that user could set the parameter value. does that make sense?

Reply:
alternatively, I can do as you suggested, see the updated